### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by daralynnrhode**
# ReadMe update 

## Overview

Additions made to the ReadMe that will link the page to the data-access api webpage. Also updated any out of date outputs. 

## Updated Files

- ReadMe

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,8 @@
 
 This lightweight  Python package allows users to download, query, and upload data from the IMAP Science Data Center (SDC).
 
+For more information and detailed instructions see https://imap-processing.readthedocs.io/en/latest/data-access-api/index.html
```
Comment (greglucas) [inline README.md:8]: I don't think markdown should use bare urls.
```suggestion
For more information see the [detailed instructions](https://imap-processing.readthedocs.io/en/latest/data-access-api/index.html).
```

File: README.md [hunk 2]
```diff
@@ -14,19 +16,28 @@ pip install imap-data-access
 imap-data-access -h
 ```
 
+### Base Command Arguments
+```bash
+$ imap-data-access query # or
+$ imap-data-access download # or
+$ imap-data-access upload
+```
+Run '-h' with any base command for more information
```
Comment (greglucas) [inline README.md:25]:
```suggestion
Add the '-h' help flag with any base command for more information
```

File: README.md [hunk 3]
```diff
@@ -40,6 +51,8 @@ $ imap-data-access query --start-date 20240101 --end-date 20241231 --output-form
 
 Download a level 0 SWE file on 2024/01/05
 
+Note: Make sure the directory configuration is correct, see Data Directory section for more information.
```
Comment (greglucas) [inline README.md:54]: You can add a warning/tip admonition here. https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts

Also, if you are indicating a user should look at a section, it is helpful to link out to that section directly.
```suggestion
> [!NOTE]
> Make sure the directory configuration is correct, see the [Data Directory section](https://github.com/IMAP-Science-Operations-Center/imap-data-access#data-directory) for more information.
```

**Review by greglucas — COMMENTED** [review]
Some minor suggestions, but overall looks great.

**Review by greglucas — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -14,19 +16,28 @@ pip install imap-data-access
 imap-data-access -h
 ```
 
+### Base Command Arguments
+```bash
+$ imap-data-access query # or
+$ imap-data-access download # or
+$ imap-data-access upload
+```
+Add the '-h' help flag with any base command for more information
```
Comment (bourque) [inline README.md:25]: Great idea to include the bear-bones usage here for quick reference. I think between this and the link to the more detailed instructions is a good balance.

**Review by bourque — APPROVED** [review]
Looks good!

**Pull request merged**